src/Spago/Command/Init.purs

```diff
@@ -234,10 +284,10 @@ pursReplFile = { name: ".purs-repl", content: "import Prelude\n" }
 -- ERROR TEXTS -----------------------------------------------------------------
 
 foundExistingProject :: FilePath -> String
-foundExistingProject path = "Found a " <> show path <> " file, skipping copy."
+foundExistingProject path = "Found a " <> path <> " file, skipping copy."
 
 foundExistingDirectory :: FilePath -> String
-foundExistingDirectory dir = "Found existing directory " <> show dir <> ", skipping copy of sample sources"
+foundExistingDirectory dir = "Found existing directory " <> dir <> ", skipping copy of sample sources"
 
 foundExistingFile :: FilePath -> String
-foundExistingFile file = "Found existing file " <> show file <> ", not overwriting it"
+foundExistingFile file = "Found existing file " <> file <> ", not overwriting it"
```

Comment:
Restored.

Reply:
Initially I found this while manually creating a fixture (I like to do that sometimes to avoid missing subtle details) and assumed it was a mistake: seeing `FilePath` instead of `String`, not realizing it's a type alias, and assuming it needs a `show` call to be able to concat with other strings. 

I think it's unstable to rely on `show` to add quotes. It's only accidental that `show` does this. I think if quotes are desired, they should be explicit: `"found \"" <> dir "\", not overwriting". Or: `"found " <> quote dir <> ", not overwriting".

Reply:
Oh yes, I remember now why I removed them. On Windows it comes out with double slashes, because `show "foo\\bar" == "foo\\\\bar"`

Reply:
I'll replace with explicit quoting.

Reply:
Wonderful, explicit quoting is much better in any case 🙂